prbt_hardware_support/include/prbt_hardware_support/sto_state_machine.h

```diff
@@ -335,28 +341,28 @@ class StoStateMachine_ : public msm::front::state_machine_def<StoStateMachine_>
   // Transitions //
   /////////////////
 
-  struct transition_table : mpl::vector<
-  //  Start                       Event          Target                      Action         Guard
-  // +---------------------------+--------------+---------------------------+--------------+----------+
-  Row< RobotInactive             , sto_updated  , Enabling                  , recover_start, sto_true >,
-  Row< RobotInactive             , sto_updated  , none                      , none         , sto_false>,
-  Row< Enabling                  , sto_updated  , none                      , none         , sto_true >,
-  Row< Enabling                  , sto_updated  , StopRequestedDuringEnable , none         , sto_false>,
-  Row< Enabling                  , recover_done , none                      , unhold_start , none     >,
-  Row< Enabling                  , unhold_done  , RobotActive               , none         , none     >,
-  Row< StopRequestedDuringEnable , sto_updated  , none                      , none         , none     >,
-  Row< StopRequestedDuringEnable , recover_done , Stopping                  , halt_start   , none     >,
-  Row< StopRequestedDuringEnable , unhold_done  , Stopping                  , hold_start   , none     >,
-  Row< RobotActive               , sto_updated  , none                      , none         , sto_true >,
-  Row< RobotActive               , sto_updated  , Stopping                  , hold_start   , sto_false>,
-  Row< Stopping                  , sto_updated  , EnableRequestDuringStop   , none         , sto_true >,
-  Row< Stopping                  , hold_done    , none                      , halt_start   , none     >,
-  Row< Stopping                  , halt_done    , RobotInactive             , none         , none     >,
-  Row< EnableRequestDuringStop   , halt_done    , Enabling                  , recover_start, none     >,
-  Row< EnableRequestDuringStop   , sto_updated  , Stopping                  , none         , sto_false>,
-  Row< EnableRequestDuringStop   , sto_updated  , none                      , none         , sto_true >
-  // +---------------------------+--------------+---------------------------+--------------+----------+
-  > {};
+  struct transition_table
+    : mpl::vector<
+          //  Start                       Event          Target                      Action         Guard
+          // +---------------------------+--------------+---------------------------+--------------+----------+
+          Row<RobotInactive, sto_updated, Enabling, recover_start, sto_true>,
```

Comment:
Is it possible to keep the formatting of such structures, for readability reasons?